user-apps/demo-pdf/default.nix

```diff
@@ -0,0 +1,28 @@
+# Copyright 2022-2023 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+{
+  stdenv,
+  pkgs,
+  lib,
+  ...
+}:
+stdenv.mkDerivation {
+  name = "demo-pdf";
+
+  src = ./Whitepaper.pdf;
+  phases = ["installPhase"];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp ${./Whitepaper.pdf} $out/Whitepaper.pdf
```

Comment:
As you are doing full mkDerivation, you could say `cp $src $out/Whitepaper.pdf` instead